### DISCUSSION TITLE
page not working

### DISCUSSION THREAD

**Discussion by pablob5** [community/community#132922] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I had a running page https://cognitivas.github.io/ but after I forked some repos and created repos it is not working anymore. No idea why, could you help me?
Best
Pablo

**Comment by knarrff** [comment]
Hi @pablob5 

You have given very little information for anyone to go on, too little I fear.

None of the 5 public repos on your account have pages enabled and there is no public repo in your account with the name cognitivas. Without a look at the repo it would be hard to tell what is wrong. In theory, actions to other repos should not influence pages of one repo, at least not if those do not have pages enabled. But also here: without knowing what it was you did it is hard to say anything.

**Comment by pablob5** [comment]
Hi Frank, thanks for your reply, and my apologies for the little
information. I am not using my personal account, I am using
https://github.com/cognitivas for the page.
The repo is this https://github.com/cognitivas/cognitivas.github.io

**Reply by knarrff** [reply]
Hi

I guess https://github.com/cognitivas/cognitivas.github.io must be private: I do get a 404 (which is normal for private repos where you do not have access). Which means there is still very little for anyone to go on.
